mscore/editstyle.cpp

```diff
@@ -716,7 +722,8 @@ const std::map<ElementType, EditStylePage> EditStyle::PAGES = {
     { ElementType::CLEF,                &EditStyle::PageClefs },
     { ElementType::KEYSIG,              &EditStyle::PageAccidentals },
     { ElementType::MEASURE,             &EditStyle::PageMeasure },
-    { ElementType::REST,                &EditStyle::PageMeasure },
+    { ElementType::REST,                &EditStyle::PageRests },
+    { ElementType::MMREST,              &EditStyle::PageRests },
```

Comment:
So there is no page for mmrests after all?

Reply:
There is! This is it.